README.md

```diff
@@ -672,6 +673,29 @@ fs.mkdir('/home', function(err) {
 });
 ```
 
+#### fs.access(path, [mode], callback)<a name="access"></a>
+
+Tests a user's permissions for the file or directory supplied in `path` argument. Asynchronous [access(2)](http://pubs.opengroup.org/onlinepubs/009695399/functions/access.html). Callback gets no additional arguments. The `mode` argument can be:
+
+* `F_OK`: Test for existence of file.
+* `R_OK`: Test whether the file exists and grants read permission.
+* `W_OK`: Test whether the file exists and grants write permission.
+* `X_OK`: Test whether the file exists and grants execute permission.
+
+NOTE: Filer allows for, but ignores the optional `mode` argument used in node.js.
+
+Example:
+
+```javascript
+/*
+* Given the following file exists:
+* '/myfile'
+*/
+ fs.access('/myfile', fs.constants.F_OK, function (error) {
```

Comment:
Why don't we steal the docs from node here instead:

```
// Check if the file exists in the current directory.
fs.access(file, fs.constants.F_OK, function(err) {
  console.log(`${file} ${err ? 'does not exist' : 'exists'}`);
});
```

It's a nice example showing how to interpret the presence of `err`